<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Codys-Mac.local" ignored="0" name="Command line test 0e1c17af-00e7-4682-a524-fc5eb4544d86" tests="2" failures="2" timestamp="04 Apr 2018 13:34:40 GMT" time="614.538" errors="0">
  <testcase name="prankcaller" time="155.129" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.prankcaller(Tests.java:239)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- prankcaller -->
  <testcase name="launchApp" time="284.594" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.launchApp(Tests.java:434)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
</testsuite> <!-- Command line test 0e1c17af-00e7-4682-a524-fc5eb4544d86 -->
